packages/jspsych/src/modules/randomization.ts

```diff
@@ -1,18 +1,14 @@
 import rw from "random-words";
-import seedrandom from "seedrandom";
+import seedrandom from "seedrandom/lib/alea";
 
 /**
  * Uses the `seedrandom` package to replace Math.random() with a seedable PRNG.
  *
  * @param seed An optional seed. If none is given, a random seed will be generated.
  * @returns The seed value.
  */
-export function setSeed(seed?: string) {
-  if (!seed) {
-    const prng = seedrandom();
-    seed = prng.int32().toString();
-  }
-  seedrandom(seed, { global: true });
+export function setSeed(seed: string = seedrandom().int32().toString()) {
```

Comment:
@jodeleeuw `seedrandom().int32().toString()` always returns the same string, since no seed is provided to `seedrandom()`. I guess your initial intention was to be independent of the possibly seeded `Math.random()`?

Reply:
Yep, that was my goal. I thought -- and I don't remember why I thought this -- that calling `seedrandom()` without a seed  randomly seeded it.

I'm fine to use `Math.random()` here instead.